scripts/build-all-apps.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+cd "$(dirname $0)"
+
+BUILD_SCRIPTS=(
+    "./../apps/gen-ext/build.sh"
+    "./../apps/kv-store/build.sh"
+    "./../apps/only-peers/build.sh"
+)
+
+run_script() {
+    local script="$1"
+    echo "Executing $script..."
+
+    # Check if the build.sh script exists and is executable
+    if [ -f "$script" ] && [ -x "$script" ]; then
+        "$script"
+        echo "$script executed successfully."
+    else
+        echo "Error: $script does not exist or is not executable."
+        exit 1
+    fi
+}
+
+# Iterate over each script in the array and run it in the background
+for script in "${BUILD_SCRIPTS[@]}"; do
+    run_script "$script" &
```

Comment:
overlapping stderr will make it tricky to determine which project caused an issue

Reply:
What do you suggest then? To not build them in parallel?

Reply:
I can also only show the result.

Reply:
> To not build them in parallel?

yeah

Reply:
Now when I have removed logs and just show execution output it is easy to identify where was the issue. 

We should optimize overall execution time as much as possible.

Reply:
Hm, I don't think we get an edge building at the same time because they write to the same target folder, which gets locked for every build. What do you observe on your end? Does it go any faster?